assets/sass/patterns/molecules/teaser.scss

```diff
@@ -25,47 +25,50 @@
 }
 
 .teaser__context_label_item {
-  display: inline;
+  @include margin(12, "bottom");
+  display: inline-block;
   list-style-type: none;
-  @include nospace();
-
-  & .teaser__context_label:after {
-    content: ", ";
-  }
-
-  &:last-child {
-    & .teaser__context_label:after {
-      content: "";
-    }
-  }
-
 }
 
 .teaser__context_label {
   // content-label rather than subject-label as not it's not always subjects in here
-  @include label-content-typeg();
+  @include font-size-and-vertical-height(12);
+  @include margin(12, "right");
+  @include padding(5 10 3 10);
+  background-color: #edeff4;
+  border-radius: 3px;
+  font-family: $font-secondary;
+  font-weight: bold;
 }
 
 // Extra specificity to override normalised style
 .teaser .teaser__context_label_link {
   text-decoration: none;
 
   .teaser__context_label {
+    background-color: #ebf2fa;
     color: $color-primary;
   }
 
 }
 
 .article-secondary-column .teaser .teaser__context_label {
-  color: $color-text-secondary;
+  @include font-size-and-vertical-height(12);
+  @include margin(12, "right");
+  @include padding(5 10);
+  background-color: #edeff4;
+  border-radius: 3px;
+  font-weight: bold;
+  color: #000;
+  display: inline;
 }
 
-.teaser__context_label_link:after {
-  content: ", ";
+.teaser .teaser__context_label_link:hover {
+  text-decoration: none;
 }
 
-.teaser__context_label_link:last-child:after {
-  content: "";
+.article-secondary-column .teaser .teaser__context_label {
```

Comment:
Can we break this down if possible as we do not want specificity if it cannot be helped.